ui/src/node_modules/app/theme.js

```diff
@@ -1,3 +1,3 @@
 export default {
-	colorMain: 'rgb(16, 174, 249)' // --color-main
+	colorMain: '#333' // --color-main
```

Comment:
I see now that we need to add `colorMainLighter` too as the sidebar `border-right` is still blueish